tools/interop_matrix/client_matrix.py

```diff
@@ -269,6 +269,7 @@ def __init__(self, patch=[], runtimes=[], testcases_file=None):
             ('v1.40.1', ReleaseInfo()),
             ('v1.41.0', ReleaseInfo()),
             ('v1.42.0', ReleaseInfo()),
+            ('v1.42.1', ReleaseInfo()),
```

Comment:
Remove v1.42.0; we just test with the most recent patch release of each minor version.